src/main/fc/rc_controls.c

```diff
@@ -145,20 +155,18 @@ void processRcStickPositions(throttleStatus_e throttleStatus)
         if (rcDelayCommand < 250)
```

Comment:
Is there a way to express this magic value in terms of time?

Reply:
Seems like this is just some sort of overflow protection, I wonder why 250 and not 255.